### PR TITLE
Support tab context menu

### PR DESCRIPTION
**Pull request by piroor**
I mainly use tab context menu instead of page context menu, so it is useful if the feature is available on  tabs. How about that?

### CODE REVIEW

**Comment by asamuzaK** [comment]
Thanks for the PR.
But in the context of tabs, it may not be suitable to show "Share page with [SNS]".
It would be better to add new tab context menu which shows "Share tab with [SNS]".

* Add new constant `SHARE_TAB = "shareTab";`
* Add `else if (id.startsWith(SHARE_TAB)) {}` in getSnsItemFromId
* Push `createMenuItem()` for `${SHARE_TAB}${id}` in createMenu

* Add localize messages for tabs in messages.json

If troublesome, file a feature request issue instead?
I'll take it then.

**Comment by piroor** [comment]
Thank you for reviewing, I try to do that.

File: src/js/background.js [hunk 1]
```diff
@@ -11,6 +11,7 @@
   const PATH_SNS_DATA = "data/sns.json";
   const SHARE_LINK = "shareLink";
   const SHARE_PAGE = "sharePage";
+  const SHARE_TAB = "shareTab";
```
Comment (asamuzaK) [inline src/js/background.js]: Sort please.

**Review by asamuzaK — CHANGES_REQUESTED** [review]

**Comment by piroor** [comment]
I've done changes to separate tab context menu from others.

File: src/js/background.js (hunk 1 above)
Comment (piroor) [reply src/js/background.js]: Oops, I didn't noted they are sorted. Thanks.

**Review by asamuzaK — APPROVED** [review]

**Comment by asamuzaK** [comment]
Thanks a lot!

**Pull request merged**